backend/src/test/java/com/funeat/acceptance/product/ProductAcceptanceTest.java

```diff
@@ -168,14 +165,12 @@ class 가격_기준_오름차순으로_카테고리별_상품_목록_조회 {
                 final var 상품2 = 단일_상품_저장(상품_삼각김밥_가격4000원_평점4점_생성(카테고리));
                 final var 상품3 = 단일_상품_저장(상품_삼각김밥_가격2000원_평점3점_생성(카테고리));
 
-                final var 예상_응답_페이지 = 응답_페이지_생성(총_데이터_개수(3L), 총_페이지(1L), 첫페이지O, 마지막페이지O, FIRST_PAGE, PAGE_SIZE);
-
                 // when
-                final var 응답 = 카테고리별_상품_목록_조회_요청(카테고리_아이디, 가격_오름차순, FIRST_PAGE);
+                final var 응답 = 카테고리별_상품_목록_조회_요청(카테고리_아이디, 가격_오름차순, 0L);
 
                 // then
                 STATUS_CODE를_검증한다(응답, 정상_처리);
-                페이지를_검증한다(응답, 예상_응답_페이지);
+                다음_페이지_유무를_검증한다(응답, false);
```

Comment:
```suggestion
                다음_페이지_유무를_검증한다(응답, 마지막페이지X);
```

PageFixture에 `마지막페이지X`, `마지막페이지O`가 있어서 이거랑 나머지도 적용하면 좋아보여요